src/database/DataTypes/TransactionBatch.js

```diff
@@ -185,6 +185,7 @@ TransactionBatch.schema = {
     type: { type: 'string', optional: true },
     linkedTransaction: { type: 'Transaction', optional: true },
     location: { type: 'Location', optional: true },
+    doses: 'double?',
```

Comment:
Was just copying the `numberOfPacks` definition - hoping there is either a reason and just following that